### PR TITLE
docs: fix links, fix outdated documentation.

### PR DESCRIPTION
**Pull request by braddialpad**
## Description

Fixed any broken links and outdated information in CONTRIBUTING.md and README.md within the individual repos.

Still some remaining work to do on documentation here. RELEASING.md will need to be reworked for the new release script. I know Julio is still doing some work on this related to alpha/beta releases so I will hold off until that is complete. We should probably also move documentation that is relevant to ALL repos into the root contributing.md or readme.md.

### CODE REVIEW

**Comment by francisrupert** [comment]
Should all the `Storybook` links be changed to `dialtone.dialpad.com/vue`, or redirect, or...?

e.g…
https://vue.dialpad.design/?path=/story/components-avatar--default
to 
https://dialtone.dialpad.com/vue/index.html?path=/story/components-avatar--default
at
[dialtone/apps/dialtone-documentation/docs/components/avatar.md?plain=1#L7](https://github.com/dialpad/dialtone/blob/fix-links/apps/dialtone-documentation/docs/components/avatar.md?plain=1#L7)

**Comment by braddialpad** [comment]
> Should all the `Storybook` links be changed to `dialtone.dialpad.com/vue`, or redirect, or...?
> 
> e.g… https://vue.dialpad.design/?path=/story/components-avatar--default to https://dialtone.dialpad.com/vue/index.html?path=/story/components-avatar--default at [dialtone/apps/dialtone-documentation/docs/components/avatar.md?plain=1#L7](https://github.com/dialpad/dialtone/blob/fix-links/apps/dialtone-documentation/docs/components/avatar.md?plain=1#L7)

Outside the scope of this PR, but yes they will need to be.

File: .github/CONTRIBUTING.md
Comment (juliodialpad) [inline .github/CONTRIBUTING.md:1]: Nice re-wording! thanks

File: packages/dialtone-icons/README.md [hunk 1]
```diff
@@ -24,6 +24,11 @@ If you need to add icons, follow the next steps to get your icons committed.
...
 3. Commit and push your changes.
 
-## Questions?
+## Requesting features / reporting bugs
 
-If you have a question, please feel free to [open an issue](https://github.com/dialpad/dialtone-icons/issues/new).
+Requesting a feature or reporting a bug? Please do so at the below links:
+
+- [Request Feature](https://dialpad.atlassian.net/secure/CreateIssue.jspa?issuetype=10975&pid=12508)
+- [Report Bug](https://dialpad.atlassian.net/secure/CreateIssue.jspa?issuetype=1&pid=12508)
```
Comment (juliodialpad) [inline packages/dialtone-icons/README.md:32]: This is not public, is this intended?

File: packages/dialtone-tokens/.github/CONTRIBUTING.md [hunk 2]
```diff
@@ -45,14 +48,24 @@ Because our design tokens are consumed on a variety of different platforms, we o
...
+
+If you would like to build the Android package locally for debugging purposes you will need to:
+
+1. Install the Android SDK
+2. Create a file called `local.properties` in the root of dialtone-tokens
+3. Add the following to the local.properties file:
+
+```properties
+sdk.dir=PATH_TO_YOUR_ANDROID_SDK_GOES_HERE
+```
```
Comment (juliodialpad) [inline packages/dialtone-tokens/.github/CONTRIBUTING.md:65]: Nice, this was missing for so long.

File: packages/dialtone-vue3/README.md
Comment (juliodialpad) [inline packages/dialtone-vue3/README.md:1]: Isn there a way to include the README.md from dialtone-vue2 here somehow? that'd be great so we can maintain only one file.

In the case it's possible, I'll like to maintain dialtone-vue3 README.md and CONTRIBUTING.md as we're possibly moving forward with vue3 version for web-components.

File: packages/dialtone-vue3/components/input/input.vue [hunk 3]
```diff
@@ -141,7 +141,7 @@ export default {
      * Name property of the input element
      */
     name: {
-      type: String,
+      type: jwfkahjfklewjfaw,
```
Comment (juliodialpad) [inline packages/dialtone-vue3/components/input/input.vue]: What happened here? lol

giphy

**Review by juliodialpad — CHANGES_REQUESTED** [review]
Nice job @braddialpad! just have a few comments/improvements.

File: packages/dialtone-vue3/components/input/input.vue (hunk 3 above)
Comment (braddialpad) [reply packages/dialtone-vue3/components/input/input.vue]: haha oh wow, was testing something with diffs and forgot to revert that

File: packages/dialtone-icons/README.md (hunk 1 above)
Comment (braddialpad) [reply packages/dialtone-icons/README.md:32]: Yeah basically.. we have the same thing on our docsite front page.

File: packages/dialtone-vue3/README.md
Comment (braddialpad) [reply packages/dialtone-vue3/README.md:1]: hmm we could just link to it...

Comment (braddialpad) [reply packages/dialtone-vue3/README.md:1]: oh looks like I already am, but from vue3 to vue2. I'll make vue 3 the main.

Comment (braddialpad) [reply packages/dialtone-vue3/README.md:1]: alright I symlinked the readme and contributing betweeen vue 2 and vue3. Any changes made to one should also change the other from now on.

**Review by iropolo — APPROVED** [review]
Lookig good!

**Review by juliodialpad — APPROVED** [review]
Nice, thanks!

**Pull request merged**